morphic.js

```diff
@@ -2690,9 +2689,7 @@ Morph.prototype.drawOn = function (aCanvas, aRect) {
 
 Morph.prototype.fullDrawOn = function (aCanvas, aRect) {
     var rectangle;
-    if (!this.isVisible) {
-        return null;
-    }
+    if (!this.isVisible) {return; }
```

Comment:
This is an example of fixing the inconsistent return error. 

See these docs: http://eslint.org/docs/rules/consistent-return.html